### ISSUE TITLE
Cached page is not being refreshed with Yii Cache Storage

### ISSUE THREAD

**Issue by paweltatarczuk** [putyourlightson/craft-blitz#257]
### Describe the bug

Yii Cache Storage is unable to refresh existing caches due to the way cache key is computed. Even when site uri is being "refreshed" Blitz is still serving an old cached version.

Site uri passed to `buildKey` method is either a string or integer and this results with different cache keys. When site uri is retrieved from the database it is a string, when computed from request it is an integer.

### To reproduce
Steps to reproduce the behaviour:
1. Configure blitz with Yii Cache Storage
2. Populate a site uri cache
3. Try refreshing the site uri cache

### Expected behaviour
Site uri should be refreshed, re-visiting the url should show refreshed page.

### Screenshots

```
$this->buildKey(['blitz', 1, 'some-path']);
"a_unique_key62808f726704789784021c1d7ea43233"
$this->buildKey(['blitz', '1', 'some-path']);
"a_unique_key0c83080cbe99f54445d9f9001cd108f7"
```

### Versions
- Plugin version: 3.6.6
- Craft version: 3.5.5

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-blitz/commit/5cef138f7d0a08202cca74fdfe684c44081db976 for the next release. Thanks for reporting this and for the excellently written issue!!

**Comment by paweltatarczuk** [comment]
Thank you for a quick fix, looks fine!

**Comment by bencroker** [comment]
Released in version 3.6.9.
